templates/zerver/help/add-an-emoji-reaction-to-a-message.md

```diff
@@ -0,0 +1,44 @@
+# Add an emoji reaction to a message
+
+![Emoji reaction](/static/images/help/emoji-react.png)
```

Comment:
Looking at user docs, we usually don't start them with an image. I think it might be a bit confusing, what do you think?